tasks/keyInfraSetters.ts

```diff
@@ -0,0 +1,41 @@
+import { task } from "hardhat/config";
+import { TaskArguments } from "hardhat/types";
+import { AccessTokenVerifier } from "../src/types";
+
+task("rotate:IntermediateKey")
+  .addParam("verifiercontract", "Address of the Access Token Verifier Contract")
+  .addParam("newintermediate", "Address of the new intermediate key")
+  .setAction(async function (taskArguments: TaskArguments, { ethers }) {
+    const newIntermediateKey = taskArguments.newintermediate;
+    if (!newIntermediateKey || newIntermediateKey === ethers.constants.AddressZero) {
+      throw new Error("Invalid new intermediate key");
+    }
+
+    const accessTokenVerifier = <AccessTokenVerifier>(
+      await ethers.getContractAt("AccessTokenVerifier", taskArguments.verifiercontract)
+    );
+
+    const tx = await accessTokenVerifier.rotateIntermediate(taskArguments.newintermediate);
```

Comment:
I think we might need to perform a `await tx.wait()` here before we can be sure the tx succeeded. This might also need to be wrapped in a `try-catch` for errors that could be unauthorised calling. Same with the rotation of Issuer key.